ArcadiaTeamsBot/Infrastructure/Bot.cs

```diff
@@ -39,7 +40,7 @@ protected override async Task OnMembersAddedAsync(IList<ChannelAccount> membersA
                 if (member.Id != turnContext.Activity.Recipient.Id)
                 {
                     await turnContext.SendActivityAsync(
-                        $"Welcome to ArcadiaBot, {member.Name}. {WelcomeText}",
+                        String.Format(WelcomeText, member.Name),

```

Comment:
What did I notice about using of built-in types and aliases? R# should be correctly configured to do this automatically.

Reply:
Fixed